lib/send_sms.rb

```diff
@@ -0,0 +1,21 @@
+# Download the twilio-ruby library from twilio.com/docs/libraries/ruby
+require 'twilio-ruby'
+
+class SendSMS
+  def initialize
+    account_sid = ENV['TWILIO_SID']
```

Comment:
Great idea using external variables to store your account details. I am curious to know what dependency you used for this? Was it dotenv?

Reply:
Thanks! I used [this model from the precourse](https://makersacademy.teachable.com/courses/256825/lectures/3989201):

Put your secret key in an environment variable and then read it from your code when you need it. For example:

`export SECRET_KEY=12345abcde`

Then, in your Ruby code, read the value:

`secret_key = ENV['SECRET_KEY']`

This way you'll be able to share your code with anyone and upload it to a GitHub repository without compromising your keys.